keep/api/routes/incidents.py

```diff
@@ -348,6 +352,36 @@ def delete_incident(
     return Response(status_code=202)
 
 
+@router.post("/merge", description="Merge incidents", response_model=IncidentDto)
+def merge_incidents(
+    command: MergeIncidentsCommandDto,
+    authenticated_entity: AuthenticatedEntity = Depends(
+        IdentityManagerFactory.get_auth_verifier(["write:incident"])
+    ),
+) -> Response:
+    tenant_id = authenticated_entity.tenant_id
+    logger.info(
+        "Merging incidents",
+        extra={
+            "source_incident_ids": command.source_incident_ids,
+            "destination_incident_id": command.destination_incident_id,
+            "tenant_id": tenant_id,
+        },
+    )
+
+    try:
+        updated_incident = merge_incidents_to_id(
+            tenant_id,
+            command.source_incident_ids,
+            command.destination_incident_id,
+            authenticated_entity.email,
+        )
+        updated_incident_dto = IncidentDto.from_db_incident(updated_incident)
+        return updated_incident_dto
+    except DestinationIncidentNotFound as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Comment:
You could actually just inherit DestinationIncidentNotFound from HTTPException and call merge_incidents_to_id without try/except here to let fastapi error handling to the magic. Not critical

Reply:
>just inherit DestinationIncidentNotFound from HTTPException
yes, but it will link the service layer with view layer. e.g. db.py will be refactored to incidents_service some day and this method could be used in non-routes methods, therefore I propose to keep it not linked.